README.md

```diff
@@ -124,7 +125,7 @@ calculateWork :: Time Hour  -- type synonym for 'Time HourUnit'
               -> (Time WorkWeek, Time WorkDay)
```

Comment:
Could you also delete comment about type synonym on line above? I forgot to delete it earlier...